src/pages/order/components/NavBar.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components'
+import { THEME } from '../../../reusable-ui/design-system'
+import { CrazeeBurgerLogo } from '../../login/components/CrazeeBurgerLogo'
+import { RightSideNavbar } from './RightSideNavbar'
+
+type NavBarProps = {
+  firstName: string | undefined
```

Comment:
```suggestion
  firstName?: string
```

est-ce que ces deux écritures sont équivalentes pour toi ?

Reply:
Non c'est deux choses différentes 👍